src/rpc/protocol.cpp

```diff
@@ -21,11 +21,18 @@
  *
  * 1.0 spec: http://json-rpc.org/wiki/specification
  * 1.2 spec: http://jsonrpc.org/historical/json-rpc-over-http.html
+ *
+ * By passing -strictjsonrpcspec one can force Bitcoin to speak strict 2.0.
+ *
  */
 
+bool fStrictJSONRPCSpec = false;
```

Comment:
Should this have some kind of `g` prefix given it's a global? Also, may want to use snake_case.